mock/datasets/no2.data.mdx

```diff
@@ -99,11 +99,11 @@ layers:
         - "#461070"
         - "#050308"
     analysis:
-      metrics:
-        - min
-        - max
-        # dummy value to make sure non existent values are sagfely discarded
-        - non-existent
+      # metrics:
+      #   - min
+      #   - max
+      #   # dummy value to make sure non existent values are sagfely discarded
+      #   - non-existent
```

Comment:
remove?

Reply:
I commented out  this part for testing; I reverted this change.